### PR TITLE
move and symlink readme

### PR DESCRIPTION
**Pull request by alangenfeld**
to have the `README.md` published in the `dagster` package and available to produce the long description on pypi, include it in the package data.

to deal with this, ive moved the root README in to the dagster directory and put a symlink at the repo root

### How I Tested These Changes

view the README in github
install the package via git and ensure the longer descrption works

### CODE REVIEW

**Comment by alangenfeld** [comment]
Current dependencies on/for this PR:
* master
  * **PR #9430** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9430" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/9430?utm_source=stack-comment).

**Comment by alangenfeld** [comment]
install seemed to work as well, I see the readme contents:

https://gist.github.com/alangenfeld/d0cd16c7aa043849e2c5e94c7c29f03f

**Review by sryza — APPROVED** [review]

File: python_modules/dagster/setup.py [hunk 1]
```diff
@@ -45,15 +45,6 @@ def get_version() -> str:
             "Operating System :: OS Independent",
         ],
         packages=find_packages(exclude=["dagster_tests*"]),
-        package_data={
-            "dagster": [
-                "dagster/core/storage/event_log/sqlite/alembic/*",
-                "dagster/core/storage/runs/sqlite/alembic/*",
-                "dagster/core/storage/schedules/sqlite/alembic/*",
```
Comment (alangenfeld) [inline python_modules/dagster/setup.py:52]: there are wrong so i assume this doesn't actually effect anything

**Review by rexledesma — APPROVED** [review]
looks good to me the readme is being captured in the package now, thanks

**Pull request merged**